draft-ietf-rtcweb-jsep.xml

```diff
@@ -446,7 +446,9 @@ setRemote(ANSWER) |                                   |
           stopped and disassociated from m= sections. An RtpTransceiver
           is never associated with more than one m= section, and once a
           session description is applied, a m= section is always
-          associated with exactly one RtpTransceiver.</t>
+          associated with exactly one RtpTransceiver. An RtpTransceiver
+          is said to be associated with an m= section if and only if
+          its mid attribute is equal to the MID of that m= section.</t>
```

Comment:
Is this in fact true? Note that we only generate MIDs if the other side provided one:

```
   o  If and only if present in the offer, an "a=mid" line, as specified
      in [RFC5888], Section 9.1.  The "mid" value MUST match that
      specified in the offer.
```